tools/development_tools/taxon_importer/taxon_importer.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class TaxonImporter
+  REQUESTED_FIELDS = %w(id name rank rank_level iconic_taxon_id is_active ancestry).freeze
+
+  attr_reader :taxon_id
+
+  class << self
+    # @param [Integer] taxon_id Remote taxon ID
+    # @return [Integer] taxon_id Locally created taxon ID
+    def import( taxon_id: )
+      new( taxon_id: taxon_id ).import
+    end
+  end
+
+  def initialize( taxon_id: )
+    @taxon_id = taxon_id
+  end
+
+  # Import will 'De-Limpkinize'
```

Comment:
Haha, love this. Delimpkinize shall be the name of the Limp Bizkit cover band that will never form.